### PR TITLE
Jenkins parameter bug

### PR DESCRIPTION
**Pull request by janetchen1**
API-1383

Fixed some string manipulation so that running "jenkins test-deploy-job -p ref=origin/master" passes, with parameter accepted. Notification of build failure, even after successful queuing, will be handled in API-1384

### CODE REVIEW

File: plugins/datadog.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+require('dotenv').config({silent: true});
+
+var core = require('../core.js');
+const ds = require('../datastore.js');
+
+function isCallable (text) {
+    return text.includes('datadog');
+}
+
+function executePlugin (channel, callback, text, user) {
```
Comment (jblanchette) [inline plugins/datadog.js:104]: One style suggestion for this function:  this has multiple nested blocks and a healthy amount of lines in the function.  This is a good candidate for breaking up the functionality into smaller isolated functions

**Comment by janetchen1** [comment]
Current status: solving the string-parameter bug breaks the code for bool-parameters. Working on a fix now. [IN PROGRESS]

**Comment by janetchen1** [comment]
As of latest commit, bot is able to run Jenkins tasks on command (API-1361) and monitor Jenkins builds that are launched (API-1384), but git parameters still not working (API-1383)

**Comment by janetchen1** [comment]
Should be ready for merge, though paper trail integration may not be 100% correct

**Pull request merged**